doc/model/train-energy-hessian.md

```
@@ -0,0 +1,50 @@
+```
+
+The options `start_pref_e`, `limit_pref_e`, `start_pref_f`, `limit_pref_f`, `start_pref_v`, and `limit_pref_v`,  determine the start and limit prefactors of energy, force, and virial, respectively. The calculation and definition of Hessian loss are the same as for the other terms.
+
+If one does not want to train with virial, they may set the virial prefactors `start_pref_v` and `limit_pref_v` to 0.
+
+
+## Hessian format in PyTorch
+
+In the PyTorch backend, Hessian matrices are listed in `hessian.npy` files, and the data format may contain the following files:
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)